site/content/blog/kctrl-app-commands-blog.md

```diff
@@ -0,0 +1,509 @@
+---
+title: "Getting to know App resources better with kctrl"
+slug: kcrl-app-commands-blog
+date: 2022-04-28
+author: Soumik Majumder
+excerpt: "Get better visibility into your app resources with kctrl's new app commands and enhancements to the package commands"
+image: /img/kapp-controller.svg
+tags: ['carvel', 'kapp-controller', 'gitops', 'apps', 'kctrl', 'cli']
+---
+
+Since the previous release of `kctrl`, we have been iterating over how we can help users take a closer look at what App CRs are doing on the cluster. This includes ones authored by users and those that are created as a result of package installations.
+
+As promised, the latest release of `kctrl` introduces a set of commands which allow users to observe and interact with App CRs more conveniently. `kctrl` v0.36.0 also introduces an improved status tailing experience which surfaces relevant information while waiting for resources to reconcile.
+
+# The `app` commands
+These commands aim to help users manage App CRs on the cluster better. Lets dig in and see what they look like!
+
+The user can list apps created on the cluster by running,
+```bash
+$ kctrl app list
+Target cluster 'https://34.135.209.41' (nodes: gke-cluster-sm-default-pool-28d3ddcd-gjqs, 2+)
```

Comment:
nit: Maybe replace this with localhost or something?

Reply:
Yep prolly should

Reply:
(Noticed that older blogs have IPs as well, this will be bonked off the grid when I delete my GKE cluster, so I guess it's fine?)

Reply:
I don't mind keeping it the same (I just get a bit anxious when I see IPs etc,. in the open so thought of mentioning).

Reply:
😆 I think I will replace them with a local IP